src/Doctrine/EntityClassGenerator.php

```diff
@@ -56,13 +56,14 @@ public function generateEntityClass(ClassNameDetails $entityClassDetails, bool $
         $this->generateRepositoryClass(
             $repoClassDetails->getFullName(),
             $entityClassDetails->getFullName(),
-            $withPasswordUpgrade)
+            $withPasswordUpgrade,
+            $avoidDefaultComments)
         ;
 
         return $entityPath;
     }
 
-    public function generateRepositoryClass(string $repositoryClass, string $entityClass, bool $withPasswordUpgrade)
+    public function generateRepositoryClass(string $repositoryClass, string $entityClass, bool $withPasswordUpgrade, bool $avoidDefaultComments = false)
```

Comment:
```suggestion
    public function generateRepositoryClass(string $repositoryClass, string $entityClass, bool $withPasswordUpgrade, bool $includeExampleComments = true)
```

I renamed the argument. But the REAL important thing here is that I'm reversing the argument to be "positive" instead of negative. Doing this isn't always possible, but when it *is* possible, it's good. Negatives are hard to understand:

```php
if ($includeExampleComments) {
    // add comments
}

// vs
if (!$avoidDefaultComments) {
    // add comments
}
```

Reply:
Done.